editor/src/app/header/header.component.ts

```diff
@@ -11,6 +11,7 @@ import { map } from 'rxjs/operators';
   selector: 'berta-header',
   template: `
     <header>
+      <div class="loading" *ngIf="isLoading$ | async"></div>
```

Comment:
One more thing - I would suggest changing the display property here. Since it's an element that will be toggled often, and changing display property is faster, then adding/removing an element.

Something like this: 
```html
<div class="loading" [style.display]="((isLoading$ | async) ? '' : 'none')"></div>
```

So when app is loading, will use whatever display value has been set in the stylesheet otherwise it will be `none`.